data/cube-hex-ro.mesh

```diff
@@ -0,0 +1,86 @@
+MFEM mesh v1.0
```

Comment:
Couldn't thi be just a version of `data/inline-hex.mesh` ?

Reply:
I wasn't aware of ```data/inline-hex.mesh```,  but it looks like it could be. We'd primarily been using this file for debugging purposes, and maybe @srwopschall could chime in this a bit more on any other usages it had in early on in the code development.